src/GoTrueClient.ts

```diff
@@ -1138,6 +1139,22 @@ export default class GoTrueClient {
       jwt: sessionData?.session?.access_token,
     })
   }
+  private async _challengeAndVerify(params: MFAChallengeAndVerifyParams): Promise<AuthMFAVerifyResponse> {
+    const { data: sessionData, error: sessionError } = await this.getSession()
+    if (sessionError) {
+      return { data: null, error: sessionError }
+    }
+    const {data: challengeData, error: challengeError} = await this.mfa.challenge({factorId: params.factorId})
+    if (challengeError) {
+      return { data: null, error: challengeError}
+    }
+    return await _request(this.fetch, 'POST', `${this.url}/factors/${params.factorId}/verify`, {
```

Comment:
why don't we call the `this._verify` method?

Reply:
makes sense, just replaced.